public/mobile-app/src/routes/address/+page.svelte

```diff
@@ -1,17 +1,38 @@
 <script lang="ts">
+  import { onMount } from 'svelte'
+  import { goto } from '$app/navigation'
   import { Address } from '$lib/address'
   import { type AddressFromBAN, callBAN } from '$lib/addressesFromBAN'
   import { userStore } from '$lib/state/User.svelte'
 
+  let addressFromUserStore: Address | null = $state(null)
   let timer: any
   let inputValue: string = $state('')
   let filteredAddresses: Address[] = $state([])
   let disabledButton: boolean = $state(true)
-  let addressHasError: boolean = $state(false)
+  let addressApiHasError: boolean = $state(false)
+  let addressInputHasError: boolean = $state(false)
   let selectedAddress: Address = $state<Address>(new Address())
   let hasSubmittedAddress: boolean = $state(false)
   let submittedAddress: Address = $state<Address>(new Address())
 
+  onMount(async () => {
+    if (!userStore.connected) {
+      goto('/')
+      return
+    } else {
+      addressFromUserStore =
+        userStore.connected.identity.address !== undefined
+          ? userStore.connected.identity.address
+          : null
```

Comment:
I believe this could simply be written:
```suggestion
      addressFromUserStore = userStore.connected.identity.address
```
If there's no address stored in the identity, then `addressFromUserStore` will be undefined (instead of `null`). Would that be an issue?

Reply:
Yes it's better, thanks!